src/ripple/test/jtx/Env.h

```diff
@@ -620,6 +628,37 @@ class Env
         AccountID, Account> map_;
 };
 
+//------------------------------------------------------------------------------
+
+namespace detail {
+
+inline
+void
+collect_args (std::vector<std::string>& v)
+{
+}
+
+template <class T, class... Args>
+void
+collect_args (std::vector<std::string>& v,
+    T const& t, Args const&... args)
+{
+    v.emplace_back(t);
+    collect_args(v, args...);
+}
+
+} // detail
+
+template <class T, class... Args>
+std::pair<int, Json::Value>
+Env::rpc (T const& t, Args const&... args)
+{
+    std::vector<std::string> v;
+    detail::collect_args(v, t, args...);
```

Comment:
Yep, agree with @vinniefalco - the interface should enforce that this function needs at least one parameter. `static_assert(sizeof...` is lame (but still get rid of `collect_args`)